README.rst

```diff
@@ -44,7 +37,8 @@ or for an editable `pip` installation with:
 
 
 Supported backend options are "SERIAL", "OPENMP", "THREADS", "HIP" and "CUDA". For "HIP" and "CUDA", the appropriate software stacks are required to enable compilation and subsequent use.
```

Comment:
```suggestion
Supported backend options are "SERIAL", "OPENMP", "HIP" and "CUDA". For "HIP" and "CUDA", the appropriate software stacks are required to enable compilation and subsequent use.
```

We may as well remove this too, since the threads lines from earlier are also removed.